src/BatteryStateOfChargeService.h

```diff
@@ -17,6 +17,13 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double AvgCurrent;
+    double prevCurrent;
+    double changeTime;
+    QTime prevTime;
+    QTime currentTime;
+    double AmpHours;
+    double existingCurrent;
+    double dTime;
```

Comment:
The convention is to have an underscore after private members like they're listed here.